lua/git-conflict.lua

```diff
@@ -529,10 +529,12 @@ local function setup_buffer_mappings(bufnr)
     return { silent = true, buffer = bufnr, desc = 'Git Conflict: ' .. desc }
   end
 
-  map('n', config.default_mappings.ours, '<Plug>(git-conflict-ours)', opts('Choose Ours'))
-  map('n', config.default_mappings.both, '<Plug>(git-conflict-both)', opts('Choose Both'))
-  map('n', config.default_mappings.none, '<Plug>(git-conflict-none)', opts('Choose None'))
-  map('n', config.default_mappings.theirs, '<Plug>(git-conflict-theirs)', opts('Choose Theirs'))
+  map({ 'n', 'v' }, config.default_mappings.ours, '<Plug>(git-conflict-ours)', opts('Choose Ours'))
+  map({ 'n', 'v' }, config.default_mappings.both, '<Plug>(git-conflict-both)', opts('Choose Both'))
+  map({ 'n', 'v' }, config.default_mappings.none, '<Plug>(git-conflict-none)', opts('Choose None'))
+  map({ 'n', 'v' }, config.default_mappings.theirs, '<Plug>(git-conflict-theirs)', opts('Choose Theirs'))
+  map({ 'v', 'v' }, config.default_mappings.ours, '<Plug>(git-conflict-ours)', opts('Choose Ours'))
```

Comment:
Is this correct? should this be `'V', 'v'`

Reply:
Yeah, the 'v' will remap to visual, visual line, and visual block mode